pkg/api/v1alpha1/cloudstackmachineconfig_webhook.go

```diff
@@ -37,6 +36,10 @@ func (r *CloudStackMachineConfig) ValidateCreate() error {
 		return apierrors.NewBadRequest("CloudStackProvider feature is not active, preventing CloudStackMachineConfig resource creation")
 	}
 
+	if err, fieldName, fieldValue := r.Spec.DiskOffering.Validate(); err != nil {
+		return apierrors.NewBadRequest(fmt.Sprintf("disk offering %s:%v, preventing CloudStackMachineConfig resource creation", fieldName, fieldValue))
```

Comment:
```suggestion
		return apierrors.NewBadRequest("disk offering [a:b], preventing CloudStackMachineConfig resource creation")
```

Reply:
discussed with Jia offline and reached an agreement that field name and value are needed for validateUpdate in webhook call. 

In validateCreate, a simple bad request can be returned, but for validteUpdate, field name has to be specified in exception.